### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by marikoja**
Refactoring file structure, Implementing API helpers, Cleanup, testing & site framework

### CODE REVIEW

File: frontend/cypress/support/commands.js [hunk 1]
```diff
@@ -11,12 +11,12 @@
...
 Cypress.Commands.add('login', () => {
-  const login = 'marikoa';
-  const password = 'MtXzwmLYTDq5Gucr';
+  const login = 'marikotest';
+  const password = 'aquarium';
 
   cy.request({
     method: 'POST',
-    url: `http://localhost:3001/api/v3/user/sign_in?login=${login}&password=${password}`,
+    url: `http://localhost:3001/api/v3/token/create?login=${login}&password=${password}`,
```
Comment (bjkeller) [inline frontend/cypress/support/commands.js:19]: Shouldn't this use the `baseURL`?

**Review by bjkeller — APPROVED** [review]
This looks good to me

Let's look at parameterizing the cypress configuration, but that can probably wait

File: frontend/cypress/support/commands.js (hunk 1 above)
Comment (marikoja) [reply frontend/cypress/support/commands.js]: Yes, we can definitely do that!

Comment (marikoja) [reply frontend/cypress/support/commands.js:19]: Turns out I'll need to do a bit more work to parameterize the front and back end separately.

**Pull request merged**